backfiller/backfiller/main.py

```diff
@@ -0,0 +1,200 @@
+
+# this is a prototype of the backfiller
+# lots about web services and the like I don't know
+# needs logging, exception handling and the like
+# also proper doc strings
+
+# when starting the backfiller and every few hours, backfill everything
+# more frequently, backfill the last couple hours
+# (last three hour directories so always at least two hours).
+
+import os
+import time
+import datetime
+import errno
+import uuid
+
+import requests
+
+import common
+
```

Comment:
I try to leave double-newlines between major "blocks" such as each top-level function or class, and between imports, constants, and the first function/class. So with that in mind: extra newline here, and between each function below